rita/src/rita_client/dashboard/eth_private_key.rs

```diff
@@ -0,0 +1,52 @@
+use super::*;
+use clarity::PrivateKey;
+
+pub fn get_eth_private_key(_req: HttpRequest) -> Box<dyn Future<Item = HttpResponse, Error = Error>> {
```

Comment:
this can just be Result<httpresponse, error> you're turning your synchronous result on line 20 into a future to match this return type and invoking a bunch of code you don't need to. But we still have lots of endpoints that work this way from back when Actix was too stubborn to take non-future types so it's no a huge deal. We're just trying to move away from it to simplify type signatures and such.